ocl/manage/views.py

```diff
@@ -11,6 +11,8 @@
 from tasks import find_broken_references, bulk_import, bulk_priority_import
 
 logger = logging.getLogger('oclapi')
+task_ids=set()
```

Comment:
We don't want to store ids in the app as it may fail or be restarted anytime whereas tasks may continue to be processed. It's also not good, if we ever need to scale the app to more instances.

Since the task.id is constructed of a username and uuid then I'd first check, if the logged in user is the one specified in task.id or a root and then return not authorized or check the task state.